lib/preload.js

```diff
@@ -17,10 +17,23 @@
   };
 
   window.onerror = function(msg, file, line, column, err){
-    if (err && msg.indexOf(err.stack) > -1) {
-      err.stack = err.stack + '\n  at ' + file + ':' + line + ':' + column
+    if (err) {
+      console.error((err.stack || err.message || err) + '\n  at ' + file + ':' + line + ':' + column);
```

Comment:
```suggestion
      const errString = err.stack || err.message || err
      console.error(errString + '\n  at ' + file + ':' + line + ':' + column);
```

To stay within the 80 chars